apps/web/lib/features/user-profile-plans.tsx

```
@@ -268,133 +269,149 @@ function AllPlans({ profile, currentTab = 'All Tasks' }: { profile: any; current
+																		style={{
+																			...provided.draggableProps.style,
+																			marginBottom: 8
+																		}}
+																	>
+																		<TaskBlockCard task={task} />
+																	</div>
+																)}
+															</Draggable>
+														)
+													)}
+													<>{provided.placeholder}</>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)